### PR TITLE
3924: Back button, accept agreement page

### PR DESCRIPTION
**Pull request by RuthBerry96**

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml [hunk 1]
```diff
@@ -31,6 +32,14 @@
         <div data-test-id="login-page">
             <div class="nhsuk-grid-row">
                 <div class="nhsuk-grid-column-full">
+                    <div data-test-id="go-back-section" class="nhsuk-back-link">
+                        <a data-test-id="go-back-link" class="nhsuk-back-link__link" href="/">
+                            <svg class="nhsuk-icon nhsuk-icon__chevron-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                                <path d="M8.5 12c0-.3.1-.5.3-.7l5-5c.4-.4 1-.4 1.4 0s.4 1 0 1.4L10.9 12l4.3 4.3c.4.4.4 1 0 1.4s-1 .4-1.4 0l-5-5c-.2-.2-.3-.4-.3-.7z"></path>
+                            </svg>
+                            Back to Public Browse
```
Comment (kat1906) [inline src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml:40]: are you sure about the wording here? it is not consistent with what we use elsewhere and also the user does not know it as 'Public Browse' - that is what we call it!

File: src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml [hunk 2]
```diff
@@ -31,6 +32,14 @@
         <div data-test-id="login-page">
             <div class="nhsuk-grid-row">
                 <div class="nhsuk-grid-column-full">
+                    <div data-test-id="go-back-section" class="nhsuk-back-link">
+                        <a data-test-id="go-back-link" class="nhsuk-back-link__link" href="/">
```
Comment (kat1906) [inline src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml:36]: will this href work?

File: src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml (hunk 1 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml:40]: No problem, what should it be called. E.g. the others are back to login? "Back to main page"?

File: src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml (hunk 2 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml:36]: It currently does, locally it reroutes to "localhost:8070/", we've done a similar thing in the account controller HttpGet Login method if the return url is null, which redirects them to the same route

File: src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml (hunk 1 above)
Comment (kat1906) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml:40]: now that is a very good question ... let me have a root around in the fe apps and I will get back to you

Comment (kat1906) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml:40]: ok we have `Back` and `Go back to previous page` - might be one for Josh

Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml:40]: I'll send him a message

Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml]: According to Josh & Luke they want it to be named "Back to homepage", so I have updated the code

**Pull request merged**